ManageTeamsIdentityMobileAndDesktop/README.md

```diff
@@ -0,0 +1,44 @@
+---
+page_type: sample
+languages:
+- Java
+products:
+- azure
+- azure-communication-services
+---
+
+
+# Create and manage Communication access tokens for Teams users in mobile and desktop applications
+
+This code sample walks you through the process of acquiring a Communication Token Credential by exchanging an Azure AD token of a user with a Teams license for a valid Communication access token.
+
+This sample application utilizes the [MSAL Java](https://github.com/AzureAD/microsoft-authentication-library-for-java) library for authentication against the Azure AD and acquisition of a token with delegated permissions. The token exchange itself is facilitated by the `azure-communication-identity` package.
+
+To be able to use the token for Calling, use it to initialize the `CommunicationTokenCredential` from the `azure-communication-common` library.
+
+## Prerequisites
+
+- An Azure account with an active subscription. [Create an account for free](https://azure.microsoft.com/free/?WT.mc_id=A261C142F).
+- Java Development Kit (JDK) version 8 or above.
+- An active Communication Services resource and connection string. [Create a Communication Services resource](https://docs.microsoft.com/azure/communication-services/quickstarts/create-communication-resource/).
+- Azure Active Directory tenant with users that have a Teams license.
+
+## Before running sample code
+
+1. Complete the [Administrator actions](https://docs.microsoft.com/azure/communication-services/quickstarts/manage-teams-identity?pivots=programming-language-javascript#administrator-actions) from the [Manage access tokens for Teams users quickstart](https://docs.microsoft.com/azure/communication-services/quickstarts/manage-teams-identity).
+    - Take a not of Fabrikam's Azure AD Tenant ID and Contoso's Azure AD App Client ID. You'll need the values in the following steps.
+1. On the Authentication pane of your Azure AD App, add a new platform of the mobile and desktop application type with the Redirect URI of `http://localhost`.
+1. Open an instance of Windows Terminal, PowerShell, or an equivalent command line and navigate to the directory that you'd like to clone the sample to.
+1. `git clone https://github.com/Azure-Samples/communication-services-java-quickstarts.git`
+1. Navigate to the `ManageTeamsIdentityMobileAndDesktop` directory.
+1. With the Communication Services procured in pre-requisites, add connection string, an Azure AD client ID and tenant ID in the code.
+
+## Run the code
+
+From a console prompt, navigate to the directory containing the `pom.xml` file, then execute the following node commands to run the app:
+
+1. Compile the project by using command `mvn compile`
+2. Then, build the package using command `mvn package`
+3. Run the command to execute the app `mvn exec:java -Dexec.mainClass="com.communication.quickstart.App" -Dexec.cleanupDaemonThreads=false`
```

Comment:
I'm getting this:

```
[INFO] Scanning for projects...
[INFO] 
[INFO] -------< com.communication.quickstart:communication-quickstart >--------
[INFO] Building communication-quickstart 1.0-SNAPSHOT
[INFO] --------------------------------[ jar ]---------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  0.321 s
[INFO] Finished at: 2022-09-07T12:18:21+02:00
[INFO] ------------------------------------------------------------------------
[ERROR] Unknown lifecycle phase ".mainClass=com.communication.quickstart.App". You must specify a valid lifecycle phase or a goal in the format <plugin-prefix>:<goal> or <plugin-group-id>:<plugin-artifact-id>[:<plugin-version>]:<goal>. Available lifecycle phases are: validate, initialize, generate-sources, process-sources, generate-resources, process-resources, compile, process-classes, generate-test-sources, process-test-sources, generate-test-resources, process-test-resources, test-compile, process-test-classes, test, prepare-package, package, pre-integration-test, integration-test, post-integration-test, verify, install, deploy, pre-clean, clean, post-clean, pre-site, site, post-site, site-deploy. -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/LifecyclePhaseNotFoundException
```

Reply:
which IDE are you using?

Reply:
vs code, running powershell core

Reply:
ok, will check out later

Reply:
try to execute as follow `mvn exec:java -D exec.mainClass="com.communication.quickstart.App" -D exec.cleanupDaemonThreads=false`. Already mentioned in the readme

Reply:
yes, this works!